spec/spec_helper.rb

```diff
@@ -2,3 +2,4 @@
 require 'koine/command_bus'
 
 require 'minitest/autorun'
+require 'support/objects.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.